src/plone/app/multilingual/tests/test_selector.py

```diff
@@ -930,3 +931,146 @@ def assertUrlsEqual(self, url1, url2):
         self.assertEqual(parsed_url_1[2], parsed_url_2[2])
         self.assertEqual(parsed_url_1[3], parsed_url_2[3])
         self.assertEqual(parse_qs_1, parse_qs_2)
+
+
+class TestLanguageSelectorSetLanguage(unittest.TestCase):
+    layer = PAM_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        # Set test variables
+        self.portal = self.layer['portal']
+        self.portal_url = self.portal.absolute_url()
+        self.request = self.layer['request']
+        alsoProvides(self.layer['request'], IPloneAppMultilingualInstalled)
```

Comment:
Isn't the fixture already taking care of assigning `IPloneAppMultilingualInstalled`? I think this is not needed, can you verify that?

Reply:
It should be. But if you check it, other testcases also do the alsoProvides dance.

Reply:
I verified and it is not needed.

We can do it for all tests in pam (in a later PR)